txscript/opcode.go

```diff
@@ -1278,7 +1278,7 @@ func opcodeIfDup(op *opcode, data []byte, vm *Engine) error {
 		return err
 	}
 
-	// Push copy of data iff it isn't zero
+	// Push copy of data if it isn't zero
```

Comment:
https://en.wikipedia.org/wiki/If_and_only_if

Reply:
Restore this change and please take a look again.